blueprints/add_drink.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+from flask import Blueprint, session, render_template
+from bson import ObjectId
+
+from .utils import *
```

Comment:
not a great way to import functions as it won't be obvious where certain functions are coming from. Import like you did on the next line or use `import utils`